Позволяет нам не обрабатывать ошибку в конкретном методе, а передавать Exception выше по уровню вплоть до main функции, однако в таком случае исключение не будет обработано и программа выкенет StackTrace и завершит свое выполнение.